pkg/config/loader.go

```diff
@@ -147,6 +149,39 @@ func LoadClient(cfg *Config) (kubernetes.Interface, error) {
 	return clientset, nil
 }
 
+func OutOfClusterClient() (kubernetes.Interface, error) {
+	kubeconfig := locateKubeconfig()
+	if len(kubeconfig) == 0 {
+		return nil, errors.New("Could not locate kubeconfig")
+	}
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not build config from kubeconfig")
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not make a new clientset from config")
+	}
+	return clientset, nil
+}
+
+func locateKubeconfig() string {
```

Comment:
Could we use Viper for this maybe? Might be nice to be able to pass in --config or something